src/utility.js

```diff
@@ -0,0 +1,31 @@
+export function getLocalTasks() {
+  let tasks;
+  if (window.localStorage.getItem('tasks') === null) {
+    tasks = [];
+  } else {
+    tasks = JSON.parse(window.localStorage.getItem('tasks'));
+  }
+  return tasks;
+}
```

Comment:
- [ ]  you can make it more simple like `let tasks = JSON.parse(window.localStorage.getItem('tasks')) || []`